app/models/select_facet.rb

```diff
@@ -12,6 +12,11 @@ def options
         value: allowed_value['value'],
         label: allowed_value['label'],
         id: allowed_value['value'],
+        data_attributes: {
+          track_category: "filterClicked",
```

Comment:
This string is defined in a few different places. Would it be possible to define it as a constant in one place and import it?